app/views/step_by_step_pages/index.html.erb

```diff
@@ -6,78 +6,121 @@
+              data_attributes: {
+                show: "scheduled"
+              }
+            },
+            {
+              text: "Unpublished changes",
+              data_attributes: {
+                show: "unpublished_changes"
+              }
+            }
+          ]
+        } %>
+      </div>
     </div>
   </div>
-</div>
 
-<table class="table table-striped" data-module="filterable-table">
-  <thead>
-    <tr>
-      <th colspan="4">
-        <div class='filter-control'>
-          <%= render partial: 'govuk_admin_template/table_filter_form', locals: { placeholder: 'Search for a step by step page here' } %>
-        </div>
-      </th>
-    </tr>
-    <tr>
-      <th>Title</th>
-      <th>Actions</th>
-      <th>Slug</th>
-      <th class="text-right">Status</th>
-    </tr>
-  </thead>
+  <div class="govuk-grid-column-three-quarters">
+    <table class="gem-c-table govuk-table govuk-table--sortable step-by-step-list__table">
+      <thead class="govuk-table__head">
+        <tr>
+          <th class="govuk-table__header" scope="col">Title</th>
+          <th class="govuk-table__header" scope="col">Slug</th>
+          <th class="govuk-table__header" scope="col">Status</th>
+        </tr>
+      </thead>
 
-  <tbody>
-    <% @step_by_step_pages.each do |step_by_step_page| %>
-      <tr data-status="<%= step_by_step_page.status[:name] %>">
-        <th>
-          <%= step_by_step_page.title %>
-        </th>
-        <td>
-          <ul class="list-inline">
-            <li><%= link_to 'Edit', step_by_step_page %></li>
-            <li><%= link_to 'Choose navigation', step_by_step_page_navigation_rules_path(step_by_step_page) %></li>
-            <% if step_by_step_page.has_draft? %>
-                <li><%= link_to 'Preview', preview_url(step_by_step_page.slug, auth_bypass_id: step_by_step_page.auth_bypass_id) %></li>
-            <% end %>
-          </ul>
-        </td>
-        <td>
-          <%= step_by_step_page.slug %>
-        </td>
-        <td class="text-right">
-          <span class="h4">
-            <span class="label <%= step_by_step_page.status[:label_class] %>" title="<%= step_by_step_page.status[:text] %>"><%= step_by_step_page.status[:text] %></span>
-          </span>
-        </td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
+      <tbody class="govuk-table__body">
+        <% if @step_by_step_pages.present? %>
+          <% @step_by_step_pages.each do |step_by_step_page| %>
+            <tr class="govuk-table__row" data-status="<%= step_by_step_page.status[:name] %>">
+              <td class="govuk-table__cell">
+                <strong><%= link_to step_by_step_page.title, step_by_step_page %></strong>
+              </td>
+              <td class="govuk-table__cell">
+                <% if step_by_step_page.has_draft? %>
+                  <%= link_to "/#{step_by_step_page.slug}", preview_url(step_by_step_page.slug, auth_bypass_id: step_by_step_page.auth_bypass_id), target: "_blank" %>
```

Comment:
This is a nitpick, but if you add the `/` then it's a base path, not a slug and the column heading should be updated to match.

Reply:
done